src/aiortc/codecs/keypointcodec.py

```diff
@@ -0,0 +1,46 @@
+# instead of random integer, each individual coordinate needs maximum 1-2 bytes because each coordiante is 0-256 range
+import audioop
+import fractions
+from typing import List, Optional, Tuple
+
+from ..jitterbuffer import JitterFrame
+from .base import Decoder, Encoder
+
+from ..mediastreams import KeypointsFrame
+
+class KeypointsDecoder(Decoder): 
+    @staticmethod
+    def _convert(data: bytes, width: int) -> bytes:
+        pass  # pragma: no cover
+
+    def decode(self, encoded_frame: JitterFrame) -> List[KeypointsFrame]:
+        keypoints_bytearray = encoded_frame.data
+        keypoints = []
+        for i in range(0, int(len(keypoints_bytearray)/4)):
+            keypoints.append([int.from_bytes(keypoints_bytearray[4*i:4*i + 2], 'big'), int.from_bytes(keypoints_bytearray[4*i + 2:4*i + 4], 'big')])
```

Comment:
please break this line up - in fact would recommend defining intermediary variables for the int version of the bytearray variables and why the indices are what they are, so its cleaerer